pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -995,7 +995,7 @@ func (b *MasqueradeBindMechanism) createNatRulesUsingIptables(protocol iptables.
 		return err
 	}
 
-	err = b.skipForwardingForReservedPortsUsingIptables(protocol)
+	err = b.skipForwardingForReservedPortsUsingIptables(protocol, portsUsedByLiveMigration())
```

Comment:
Since now we pass the ports that need to be skipped the function does not know if they are reserved or not so I would rename this `skipForwardingForPortsUsingIptables`